src/runtime/graph/graph_runtime.cc

```diff
@@ -59,6 +60,21 @@ class GraphRuntime : public ModuleNode {
       if (op_execs_[i]) op_execs_[i]();
     }
   }
+
+  void Run_Profile() {
+    // setup the array and requirements.
+    this->SetupProfile();
+    auto start_time = std::chrono::high_resolution_clock::now();
+    for (size_t i = 0; i < op_execs_.size(); ++i) {
+      if (op_execs_[i]){
+        auto op_start = std::chrono::high_resolution_clock::now()-start_time;
```

Comment:
Style issue. Space before and after minus.

You can use pylint and cpplint to check code style.